recipes/CommonVoice/ASR/CTC/README.md

```diff
@@ -9,19 +9,22 @@ It is important to note that CommonVoice initially offers mp3 audio files at 42H
 
 # Languages
 Here is a list of the different languages that we tested within the CommonVoice dataset and CTC:
+- English
+- German
 - French
+- Italian
 - Kinyarwanda
-- English
 
 # Results
-
 | Language | CommonVoice Release | hyperparams file | LM | Val. CER | Val. WER | Test CER | Test WER | HuggingFace link | Model link | GPUs |
 | ------------- |:-------------:|:---------------------------:| -----:| -----:| -----:| -----:| -----:| :-----------:| :-----------:| :-----------:|
 | English | 2020-12-11 | train_en_with_wav2vec.yaml | No | 5.01 | 12.57 | 7.32 | 15.58 | Not Avail. | [model](https://drive.google.com/drive/folders/1tYO__An68xrM5pR1UIXzEkwzvKX2Tz2o?usp=sharing) | 2xV100 32GB |
+| German | 2022-08-16 | train_de_with_wav2vec.yaml | No | 1.90 | 8.02 | 2.40 | 9.54 | Not Avail. | [model](https://drive.google.com/drive/folders/19G2Zm8896QSVDqVfs7PS_W86-K0-5xeC?usp=sharing) | 1xRTXA6000 48GB |
```

Comment:
update the link to the speechbrain huggingface model -> https://huggingface.co/speechbrain/asr-wav2vec2-commonvoice-de

Reply:
Done!